.golangci.yml

```diff
@@ -69,3 +69,13 @@ issues:
     - EXC0003 # test/Test ... consider calling this
     - EXC0004 # govet
     - EXC0005 # C-style breaks
+  exclude-rules:
+    - path: doc_test.go
+      linters:
+        - revive
+      text: "empty-block"
+    - linters:
+        - revive
+      text: "unused-parameter"
+  max-issues-per-linter: 0
+  max-same-issues: 0
```

Comment:
These `max-*` look dangerous, I'd like to see everything there is from a linter to fix things in batches.

Reply:
Done.